README.md

```diff
@@ -474,6 +474,26 @@ Use this command to print the version of elastic-package that you have installed
 
 
 
+## Elastic Package profiles
+
+The `profiles` subcommand allows to work with different configurations. By default,
+`elastic-package` uses the "default" profile. Other profiles can be created with the
+`elastic-package profiles create` command. Once a profile is created, it will have its
+own directory inside the elastic-package data directory. Once you have more profiles,
+you can change the default with `elastic-package profiles use`.
+
+You can find the profiles in your system with `elastic-package profiles list`.
+
+You can delete profiles with `elastic-package profiles delete`.
+
+Each profile can have a `config.yml` file that allows to persist configuration settings
+that apply only to commands using this profile.
+
+The following settings are available per profile:
+
+* `stack.geoip_dir` defines a directory with GeoIP databases that can be used by
```

Comment:
It's my understanding from this that the value should point to a directory outside the profile dir. Is that correct? If it is, it probably should be noted here.

Reply:
Not mandatory, but definitely recommended, added a line about this. Thanks.